package.json

```diff
@@ -22,7 +22,8 @@
     "webpack-prod": "NODE_ENV=production webpack --mode=production",
     "webpack-watch": "webpack --watch --mode=development",
     "test": "jest --runInBand",
-    "test:ci": "yarn test --coverage"
+    "test:ci": "yarn test --coverage",
+    "minimize-img": "./scripts/process-images.sh _site/docs/assets/img"
```

Comment:
directory passed as argument to `process-images.sh` needs to match file path in `/tmp/workspace/`

Reply:
I would ask @rsylvian how we feel about hardcoding this w/ `docs` in the path. I see this can work if we're only running it only on master branch but it creates 2 issues.

1. If we ever make changes to our S3 bucket structure, this is a hardcoded path not reliant on `JEKYLL_BASENAME`.
2. no true testability of images w/ preview branches

Reply:
Ill see if romain is able to sync on that today

Reply:
Good callout @nramabad. Would be great to avoid hardcoding the path if that's an easy lift.